src/DssSpell.t.sol

```diff
@@ -614,369 +604,426 @@ contract DssSpellTest is DssSpellTestBase {
         assertEq(arbitrumGateway.validDomains(arbDstDomain), 0, "l2-arbitrum-invalid-dst-domain");
     }
 
+    function testSparkSpell() public {
+        ACLManagerLike SPARK_ACL_MANAGER = ACLManagerLike(0xb137E7d16564c81ae2b0C8ee6B55De81dd46ECe5);
+        address SPARK_PROXY = 0x4e847915D8a9f2Ab0cDf2FC2FD0A30428F25665d;
+        address RETH = 0x62BC478FFC429161115A6E4090f819CE5C50A5d9;
+        PoolLike pool = PoolLike(0x26ca51Af4506DE7a6f0785D20CD776081a05fF6d);
+
+        // Spell is thoroughly checked in Spark repo, but just triple check the spell was cast here
+        assertEq(WardsAbstract(SPARK_PROXY).wards(address(esm)), 0);
```

Comment:
BTW RWA014 uses `address ESM                                = addr.addr("MCD_ESM");` to load up the ESM.

You can use `MCD_ESM` as the varname here and could have used it without the address() casts, which would be simpler.

Not requesting a change, just letting you know

Reply:
esm is imported from the `DssSpell.t.base.sol`
i don't see the point of reinstantiating things that are already imported tbh.